includes/class-newspack-ui.php

```diff
@@ -114,7 +114,7 @@ class="newspack-ui__snackbar__item newspack-ui__snackbar__item--<?php echo esc_a
 						>
 							<?php if ( ! $notice['autohide'] ) : ?>
 								<button class="newspack-ui__snackbar__close" aria-label="<?php esc_attr_e( 'Close', 'newspack-plugin' ); ?>" title="<?php esc_attr_e( 'Close', 'newspack-plugin' ); ?>">
-									<?php Newspack_UI_Icons::print_svg( 'closeSmall' ); ?>
+									<?php \Newspack\Newspack_UI_Icons::print_svg( 'closeSmall' ); ?>
```

Comment:
NIT: the `\Newspack\` prefix isn't necessary in this class because it shares a namespace with the `\Newspack\Newspack_UI_Icons` class, and there's also a `use Newspack\Newspack_UI_Icons;` statement at the top of the file which would allow using the class without the prefix even if they didn't share a namespace.

```suggestion
									<?php Newspack_UI_Icons::print_svg( 'closeSmall' ); ?>
```